scripts/install-worker.sh

```diff
@@ -518,7 +520,7 @@ echo vm.max_map_count=524288 | sudo tee -a /etc/sysctl.conf
 ### adding log-collector-script ################################################
 ################################################################################
 sudo mkdir -p /etc/eks/log-collector-script/
-sudo cp $TEMPLATE_DIR/log-collector-script/eks-log-collector.sh /etc/eks/log-collector-script/
+sudo cp "${TEMPLATE_DIR}/log-collector-script/eks-log-collector.sh" "/etc/eks/log-collector-script/"
```

Comment:
You don't need to quote the destination path.

Reply:
nice catch!